arex-storage-web-api/src/main/java/com/arextest/storage/web/controller/AgentRecordingController.java

```diff
@@ -96,6 +106,31 @@ public byte[] query(@RequestBody AREXMocker requestType,
     return ZstdJacksonSerializer.EMPTY_INSTANCE;
   }
 
+  @PostMapping(value = "/queryMockers")
+  @ResponseBody
+  public byte[] queryMockers(@RequestBody QueryMockRequestType requestType) {
+    try {
+      if (StringUtils.isEmpty(requestType.getRecordId())) {
+        LOGGER.warn("agent query recordId empty");
+        return ZstdJacksonSerializer.EMPTY_INSTANCE;
+      }
+
+      MDCTracer.addRecordId(requestType.getRecordId());
+      return agentWorkingService.queryMockers(requestType.getRecordId(), requestType.getFieldNames(), requestType.getCategoryTypes());
+    } catch (Exception e) {
+      LOGGER.error("query error:{} from category:{}", e.getMessage(), requestType, e);
+    } finally {
+      MDCTracer.clear();
+    }
+    return ZstdJacksonSerializer.EMPTY_INSTANCE;
+  }
+
+  @PostMapping(value = "/queryMockersWithoutZstd")
+  @ResponseBody
+  public List<AREXMocker> queryMockersWithoutZstd(@RequestBody QueryMockRequestType requestType) {
```

Comment:
There is no need to create an interface because different media types match different `MessageConverter`